src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/simplifybooleanreturn/Example1.java

```diff
@@ -0,0 +1,55 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="SimplifyBooleanReturn"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.simplifybooleanreturn;
+
+// xdoc section -- start
+class Example1 {
+
+  boolean cond;
+  int a;
+  int b;
+
+  boolean check1() {
+    if (cond) { // violation, 'Conditional logic can be removed
+      return true;
+    }
+    else {
+      return false;
+    }
+  }
+
+  // Ok, simplified version of check1()
+  boolean check2() {
```

Comment:
lets remove comment and rename method
`boolean check1Simplified() {`

please do same for "int choose2() {"

Reply:
Done